### PR TITLE
Deploy docker proxy components in both clusters

### PR DESCRIPTION
**Pull request by fgimenez**
Moves the tasks and templates required for the docker proxy components from prow role to shared-deployments, so that they are deployed on both clusters.

cc/ @rmohr @dhiller 

Signed-off-by: Federico Gimenez <fgimenez@redhat.com>

### CODE REVIEW

File: github/ci/remote-cluster/templates/mirror-proxy.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  template:
+    metadata:
+      labels:
+        app: docker-mirror-proxy
+    spec:
+      terminationGracePeriodSeconds: 180
+      serviceAccountName: "caches"
+      containers:
+      - name: mirror-proxy
+        image: fgimenez/docker-registry-proxy:v0.2.0
```
Comment (dhiller) [inline github/ci/remote-cluster/templates/mirror-proxy.yaml:19]: In the long run we should push that to kubevirtci org on quay also.

**Review by dhiller — APPROVED** [review]
/lgtm

File: github/ci/remote-cluster/templates/mirror-proxy.yaml (hunk 1 above)
Comment (fgimenez) [reply github/ci/remote-cluster/templates/mirror-proxy.yaml:19]: makes total sense :+1:

**Review by dhiller — APPROVED** [review]
/lgtm
/approve

**Review by dhiller — APPROVED** [review]
/lgtm

**Pull request merged**